model/src/test/java/org/projectnessie/error/TestNessieError.java

```diff
@@ -59,7 +62,7 @@ void fullMessage() {
                 + "foo.bar.InternalServerError\n"
                 + "\tat some.other.Class\n"
                 + "java.lang.Exception: processingException\n"
-                + "\tat org.projectnessie.error.TestNessieError.fullMessage(TestNessieError.java:");
```

Comment:
Is this change needed for concurrent runs? why?

Reply:
The test original intention was to ensure that the client-side stack trace is reflected in the full message. Could you add another (more relaxed) check to validate that `TestNessieError.fullMessage` is there?

Reply:
ah, you're right. gimme a sec

Reply:
done